src/schema.ts

```diff
@@ -43,15 +46,30 @@ export class Schema {
         }
       } else if (primitiveType === PrimitiveType.DATE) {
         type = Type.TEMPORAL;
+        // need to get correct min/max of date data
```

Comment:
> Datalib's summary method doesn't get the type correct. It considers the length of the date string when it determines the stats, so we have to manually figure out the min/max date.

This sounds like the comment you should write down.   Just saying 

>  // need to get correct min/max of date data

doesn't help much because readers don't know why you need to do it.